### ISSUE TITLE
[Feature request] Ability to determine how many rows the table legend should be before it scrolls

### ISSUE THREAD

**Issue by trevorndodds** [grafana/grafana#8292]
Currently when using the legend as a table it would occupy roughly 50% of the graph panel. It would be nice if we could control how many lines would be displayed before it would scroll. Sometimes I would like more emphasis on the graph with only a small visual/scroll of the table.
Adjusting the legend would thus increase/decrease the graph render size. You could still keep the current behavior as the default but have the ability to reduce the legend % thus growing the graph size.

**Comment by edin008** [comment]
Is this feature request being considered?

Please implement posibility to fix number of rows in legend, would help greatly in visualising dashboards in sense of putting more information on whole dashboard before page scroll.

My problem is a bit different than from trevorndodds but this feature would solve it. I want to put as many graphs as possible with exactly 4 rows in legend before page scroll but now with 4 rows in legend the graph automaticaly gets bigger making whole dashboard longer which means page scroll.

**Comment by baldm0mma** [comment]
Yep, we are considering how to implement this for next quarter, along with a number of other Legend updates.

**Comment by FelixGV** [comment]
Hi, I would also like to have the ability to control the number of lines in the legend, even expanding it to include all available lines. What is the next step to get this done?
